### ISSUE TITLE
w+b ?

### ISSUE THREAD

**Issue by alexzk1** [YoCodingMonster/OpenFreezeCenter#181]
I'm trying to write normal C++ code implementing this idea.
So I found something weird into library used:
```
 with open(EC_IO_FILE,'w+b') as file:
        file.seek(BYTE)
        file.write(bytes((VALUE,)))
```
Are you sure it should be `w+b` ? Because 

> 
> "w+"
> 
> Creates an empty file for both reading and writing.

So it should truncate content, then `seek` looks useless there.

**Comment by YoCodingMonster** [comment]
w+b for createing and writing the file
same as w+

**Comment by alexzk1** [comment]
> w+b for createing and writing the file same as w+

Yes, that what I said. If you do that with normal file it will get 0 length. So following "seek" has no sense then. debug fs may ignore "truncate" request, but that does not cancel the fact code tries to do it. Looks like bug.
